### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by exetico**
Thanks for a great script. Here is a few suggestions to the readme-file.

I did a small type error, you may want to change:
Using git it way eaiser
>
Using git **is** way eaiser

Best Regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,29 @@ Plex Update is a BASH script which simplifies the life of headless Linux Plex Me
...
 
-## 2. setting it up
+You can either just download it as a [zip file](https://github.com/mrworf/plexupdate/archive/master.zip) and unzip it on your server.
+```
+wget https://github.com/mrworf/plexupdate/archive/master.zip && unzip master.zip && mv plexupdate-master plexupdate && rm master.zip
+```
+Note that unzip is requered (`sudo apt-get install unzip`).
+
+####Using git to clone (Recommended)
+Using git it way eaiser and recommended, if you ask me.
```
Comment (mrworf) [inline README.md]: Fix the it to is and I'll approve it :)

**Comment by mrworf** [comment]
Fair update, once the spelling is fixed, I'm going to merge it

**Comment by exetico** [comment]
Thanks. 

Just fixed the spelling, and a bit extra - including a logo.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,34 @@
+![plexupdate.sh](http://i.imgur.com/ThY5Rvl.png "plexupdate")
 # plexupdate
 
-Plex Update is a BASH script which simplifies the life of headless Linux Plex Media Server users (how's that for a strange description).
+Plex Update is a BASH script which simplifies the life of headless Linux Plex Media Server users (what's that for a strange description).
```
Comment (mrworf) [inline README.md]: Actually, "How's that for a ...." is an expression and is correct. Changing it to what's that... does not make any sense. See https://www.google.com/search?q=%22how+is+that+for+a%22&oq=%22how+is+that+for+a%22

File: README.md [hunk 3]
```diff
@@ -1,18 +1,34 @@
+![plexupdate.sh](http://i.imgur.com/ThY5Rvl.png "plexupdate")
```
Comment (mrworf) [inline README.md]: LOVE IT!

**Comment by mrworf** [comment]
Almost there :) Thanks for your patience

**Comment by exetico** [comment]
Damn, that's fun. I actually googled the expression, but... i kinda did something wrong then ;-)

**Pull request merged**